src/assets/JSONLDAsset.js

```diff
@@ -0,0 +1,61 @@
+const urlJoin = require('../utils/urlJoin');
+const isURL = require('../utils/is-url');
+const Asset = require('../Asset');
+
+// A list of all attributes in a schema that may produce a dependency
+// Based on https://schema.org/ImageObject
+// Section "Instances of ImageObject may appear as values for the following properties"
+const SCHEMA_ATTRS = [
+  'logo',
+  'photo',
+  'image',
+  'thumbnail',
+  'screenshot',
+  'primaryImageOfPage'
```

Comment:
You could look through https://schema.org/URL for more URL instances. Maybe Video/MediaObject and a few others of those would be good to support too.